addons/escoria-core/game/core-scripts/esc_location.gd

```diff
@@ -1,5 +1,8 @@
 # A simple node extending Position2D with a global ID so that it can be
-# referenced in ESC Scripts.
+# referenced in ESC Scripts. Movement-based commands like walk_to_pos will
+# automatically use an ESClocation that is a child of the destination node.
+# Commands like turn_to which are not movement-based will ignore child
+# ESClocations and refer to the parent node.
```

Comment:
```suggestion
# referenced in ESC Scripts. Movement-based commands like `walk_to_pos` will
# automatically use an `ESCLocation` that is a child of the destination node.
# Commands like `turn_to`--which are not movement-based--will ignore child
# `ESCLocation`s and refer to the parent node.
```

Reply:
I'm not sure how "--" gets translated by the document generator, but I'll take your word for it that that's how it should look @BHSDuncan 😄